src/components/TemplateList/TemplateList.js

```diff
@@ -254,23 +254,12 @@ const TemplateList = ({
           const url = studyKfid
             ? `${KF_STUDY_API}/download/templates/${match.params.kfId}?file_format=${format}&template_versions=${tempList}`
             : `${KF_STUDY_API}/download/templates?file_format=${format}&template_versions=${tempList}`;
-          let anchor = document.createElement('a');
-          document.body.appendChild(anchor);
           let file = url;
           let headers = new Headers();
           headers.append('Authorization', bearerToken);
-          const fileName = `${studyKfid ? studyKfid : 'data'}_templates${
```

Comment:
Every time I try to download a template, I get a new window with `Not authorized toe download templates`. I think the logic you had before (everything you've removed) worked, and the only change needed is probably setting `anchor.download` to the fileName from the `Content-Disposition` header. Maybe something weird happened to this commit bc I don't see any `Content-Disposition` parsing anymore?